--- a/src/environment.py
+++ b/src/environment.py
@@ -74,7 +74,7 @@ def make_training_env(num_envs=1, **level_kwargs):
     if num_envs == 1:
         return prepare_env(
             retro.make(
             'SuperMarioWorld-Snes',
             state='DonutPlains5',
-            render_mode='human', # Change to 'rgb_array' when debugging finished,
+            render_mode='rgb_array',  # Use rgb_array for headless training
         ))
     else:
